data/demographic/census/index.html

```diff
@@ -59,7 +77,36 @@
 {% include abstract.html
 name="Demographic Data"
 stewards="AGRC and U.S. Census Bureau"
-abstract="This data package contains demographic information collected by the Census Bureau in 2010 and 2000. Demographic population includes population counts, household counts, ethnicity counts, etc.  Below, the U.S. Census Bureau Data is the data in the format released by the Census Bureau, with just a few modifications to make the data more user friendly. The Populated Block Areas 2010 Approximation and Populated Place Points 2010 Approximation are datasets derived by AGRC from 2010 Census data." %}
+abstract="This page contains boundary and demographic data collected by the U.S. Census Bureau for 2020, 2010 and 2000 censuses. Demographic data includes population counts, household counts, ethnicity counts, etc. The data available on this page is filtered, geographically, to the state of Utah. This data follows the format released by the Census Bureau, with just a few modifications for usability." %}
+<p>Read about <a href="{% link data/demographic/census/index.html %}#Commonly-Used Census Bureau Geography">commonly-used Census Bureau geography</a> including blocks, block groups, tracts, and places.</p>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+      <h3 id="2020Census">2020 U.S. Census Bureau Data</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+      <img class="inline-text-left outline" src="{% link images/censussmall.png %}" alt="2020 U.S. Census Bureau Data" loading="lazy" />
+      <p>The data in this section contains census blocks, census block groups, census tracts, census places, census counties, and the state boundary from the Census Bureau.</p>
+      <p>Utah’s 2020 census geographic boundaries became available in February 2021. The Census Bureau will release the first set of corresponding demographic data in September 2021 (the 2020 Census Redistricting P.L. 94-171 Summary Files). Following that release, AGRC will append the demographic data to the existing 2020 geographies served on this page.</p>
```

Comment:
not sure if this reads better or not
```suggestion
      <p>Utah’s 2020 census geographic boundaries became available in February 2021. The Census Bureau will release the first set of corresponding demographic data in September 2021 (the 2020 Census Redistricting P.L. 94-171 Summary Files). Following that release, AGRC will merge the demographic data with the existing 2020 data available now.</p>
```